src/App.js

```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+// import logo from './logo.svg';
+// import './App.css';
```

Comment:
- [ ] Kindly make sure to remove all comments in your codebase in order to keep your repo clean and professional.

Reply:
well resolved! ✔️